apiary-metastore-listener/src/main/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListener.java

```diff
@@ -145,8 +151,13 @@ private void publishEvent(
     throws MetaException {
     JSONObject json = createBaseMessage(eventType, table.getDbName(), table.getTableName());
 
+    json.put("tableLocation", table.getSd().getLocation());
+
+    json.put("hkaasParameters", getHkaasParams(table.getParameters()));
```

Comment:
can we make hkaas optional based on a environment variable ENABLE_HKAAS? all deployments may not need this information

Reply:
If its not provided the `hkaasParameter` will be empty and can be ignored.

Reply:
should be called `tableParameters`, as @massdosage  said nothing hkaas should exist in this project.

Reply:
Yeah. I have renamed it to `tableParameters`.